force-app/main/default/classes/DuplicateEnrollmentException.cls

```diff
@@ -0,0 +1,2 @@
+public class DuplicateEnrollmentException extends Exception {
```

Comment:
Please don't include these empty files inside of your commits.

Reply:
This is my custom exception. I thought we only needed to create a class that extended exception for that. Is there something else that is supposed to be in the body?